doc/gov/chapters/chapter09.md

```diff
@@ -0,0 +1,121 @@
+[<< Back](../)
+
+# 9. Adoption
+<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
+
+## Table of Contents
+* [8.1 Overview](#8.1)
+* [8.2 Transitional Plan](#8.2)
+  * [8.2.1 Rationale](#8.2.1)
+  * [8.2.2 VNF Evolution Generations](#8.2.2)
+  * [8.2.3 CNF Evolution Generations](#8.2.3)
+  * [8.2.4 Policies](#8.2.4)
+* [8.3 Adoption Strategy](#8.3)
+* [8.4 Adoption Roadmap](#8.4)
+  * [8.4.1 CNTT Roadmap](#8.4.1)
+  * [8.4.2 VNF Adoption Roadmap](#8.4.2)
+  * [8.4.3 CNF Adoption Roadmap](#8.4.3)
```

Comment:
How this relates to Adoption is a bit unclear to be. Is this the right/most intuitive word?